src/components/SidePanes/CatalogInclusionPane.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Form, Spinner } from '@edx/paragon';
+import {
+  PUBLISHED, REVIEW_BY_INTERNAL, REVIEWED, UNPUBLISHED,
+} from '../../data/constants';
+
+import DiscoveryDataApiService from '../../data/services/DiscoveryDataApiService';
+import Pane from './Pane';
+
+const CatalogInclusionPane = ({ courseUuid, subInclusion, draftStatuses }) => {
+  const [inclusion, setInclusion] = useState(subInclusion);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  const toggleInclusion = async (e) => {
+    e.preventDefault();
+    setIsLoading(true);
+    let draft = null;
+    if (draftStatuses.includes(PUBLISHED)) {
+      draft = false;
+    }
+    if (draftStatuses.includes(UNPUBLISHED)) {
+      draft = true;
+    }
```

Comment:
Putting the publisher's draft flag behavior reference for Course and Course runs below, cause I suspect we would need to update this check.
- https://github.com/openedx/frontend-app-publisher/blob/master/src/components/EditCoursePage/index.jsx#L196
- https://github.com/openedx/frontend-app-publisher/blob/master/src/components/EditCoursePage/index.jsx#L117